docs/lang-python.md

```diff
@@ -1,3 +1,112 @@
 # Using WebAssembly from Python
 
+Wasmtime can be used as a python module loader, which allows almost any
+Webassembly module to be used as a python module. This guide will go over adding
+Wasmtime to your project, and some provided examples of what can be done with
+Webassembly modules.
+
+## Prerequisites
+
+To follow this guide, you'll need
+
+ - Python 3.6 or newer
+ - The [Webassembly binary toolkit](https://github.com/WebAssembly/wabt/releases)
+ - The rust toolchain installer [rustup](https://rustup.rs/)
+
+## Getting started and simple example
+
+First, copy this example Webassembly text module into your project. It exports a
+function for calculating the greatest common denominator of two numbers.
+
+```wat
+{{#include ../examples/gcd.wat}}
+```
+
+Before we can do anything with this module, we need to convert it to the
+Webassembly binary format. We can do this with the command line tools provided
+by the Webassembly binary toolkit
+
+> wat2wasm gcd.wat
```

Comment:
I think these will want to be surrounded in three ticks (\`) instead of with a `>` because I believe that's a blockquote?

e.g.

```
wat2wasm gcd.wat
```

vs

> wat2wasm gcd.wat

Reply:
Sounds good, fixed it